src/Symfony/Component/Mailer/Bridge/Sendgrid/Tests/Http/Api/SendgridTransportTest.php

```diff
@@ -58,4 +58,51 @@ public function testSend()
 
         $mailer->send($email);
     }
+
+    public function testAttachmentEncoding()
+    {
+        $email = new Email();
+        $email->from('foo@example.com')
+            ->to('bar@example.com')
+            ->attach('Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod', 'lorem.txt');
```

Comment:
There is no line-break in this content, so this is not testing your patch.

Reply:
@stof it does, line-breaks are added by the MIME-compliant base64 encoder: https://github.com/symfony/symfony/blob/7788612c3cb63cc44b2cabb4644a7fbd75c61e97/src/Symfony/Component/Mime/Encoder/Base64Encoder.php#L39

Reply:
This is so confusing that we should add a comment before the `->attach()` method. Something like:

```php
// even it content doesn't include new lines, the Base64 encoding performed later may add them
->attach('...)
```